app/src/main/java/fr/neamar/kiss/db/DB.java

```diff
@@ -27,23 +27,28 @@ private void createTags(SQLiteDatabase database) {
         database.execSQL("CREATE INDEX idx_tags_record ON tags(record);");
     }
 
+    private void createBadges(SQLiteDatabase database) {
```

Comment:
@alexander255 @nmitsou I missed to call this method on the onCreate. Thats the reason test is failing.